src/database/adapter.js

```diff
@@ -0,0 +1,9 @@
+module.exports = env => {
+  // If we have a relational database, just grab the relational adapter
+  if(['mysql', 'pg'].indexOf(env('DATABASE_ENGINE')) !== -1) {
+    return require('./adapters/relational')(env);
+  // Otherwise, grab the specific database adapter
+  } else {
```

Comment:
Can we make this explicit? (i.e. checking that `DATABASE_ENGINE === 'mongodb'`. And maybe throwing an error if the value is not supported?

If we throw an error this is a breaking change. But we can default the value to 'pg' using `env('DATABASE_ENGINE', 'pg')`